starters/solidstart-tanstackquery-tailwind-modules/README.md

```diff
@@ -1,30 +1,87 @@
-```
+- [Overview](#overview)
+  - [Tech Stack](#tech-stack)
+  - [Included Tooling](#included-tooling)
+  - [Example Components](#example-components)
+- [Installation](#installation)
+  - [CLI](#cli)
+  - [Manual](#manual)
+- [Commands](#commands)
+- [Demo Implementation](#demo-implementation)
+
+## Overview
+
+### Tech Stack
+
+- [SolidStart](https://start.solidjs.com/getting-started/what-is-solidstart/)
+- [TanStack Query](https://tanstack.com/query/v4/docs/solid/overview)
+- [Tailwind CSS](https://tailwindcss.com/)
+
+### Included Tooling
+
+- [Vitest](https://vitest.dev/) - Test runner
+- [Storybook](https://storybook.js.org/) - Component library
+- [Typescript](https://www.typescriptlang.org/) - Type checking
+<!-- - [ESLint](https://eslint.org/) - Code linting -->
+<!-- - [Prettier](https://prettier.io/) - Code formatting -->
+
+### Example Components
+
+#### Counter `src/components/CounterExample`
+
+A small counter example that showcases how to use state in SolidJS. You can view the example on the `src/pages/Counter`.
+
+Signals can live outside of components. Each relevant component subscribes to its value by using it.
 
-## Developing
+#### Fetch `src/components/FetchExample`
 
-Once you've created a project and installed dependencies with `npm install` (or `pnpm install` or `yarn`), start a development server:
+Demonstrates how to retrieve data from a third-party API by using `createResource`.
+
+## Installation
+
+### CLI (Recommended)
 
 ```bash
-npm run dev
+npm create @this-dot/starter --kit solidstart-tanstackquery-tailwind-modules
+```
+
+- Follow the prompts to select the solidstart-tanstackquery-tailwind-modules starter kit and name your new project.
+- `cd` into your project directory and run `npm install`.
+- Run `npm run dev` to start the development server.
+- Open your browser to `http://localhost:3000` to see the included example code running.
+
+### Manual
 
-# or start the server and open the app in a new browser tab
-npm run dev -- --open
+```bash
+git clone https://github.com/thisdot/starter.dev.git
 ```
 
-## Building
+- Copy and rename the `starters/solidstart-tanstackquery-tailwind-modules` directory to the name of your new project.
+- `cd` into your project directory and run `pnpm install`.
+- Run `npm run dev` to start the development server.
+- Open your browser to `http://localhost:3000` to see the included example code running.
+
+## Commands
+
+- `pnpm run dev` - Runs the development server on localhost port 3000 with HMR
+- `pnpm run test` - Runs the test suite
+<!-- - `pnpm run storbook` - To showcase the component library -->
+- `pnpm run build` - Builds a production version of the app to deploy
+- `pnpm run serve` - Serves a production build on localhost port 4173
+<!-- - `pnpm run lint` - Uses eslint to find potential issues in the codebase -->
+<!-- - `pnpm run lint-fix` - Tries to auto-fix potential issues -->
+<!-- - `pnpm run format` - Fixes formatting issues in the codebase -->
```

Comment:
Since we would like to be package manager agnostic, this list of commands should be "pure", so we should remove all `pnpm run` in front of all the commands